src/02-Information-About-Guide.md

```diff
@@ -0,0 +1,7 @@
+## 0.2 Information About Guide
+
+This guide is made by [Patryk Adamczyk](mailto:patrykadamczyk@patrykadamczyk.net).
+
+This is PAiP Web Developer Guide version 1.
```

Comment:
Change Version to SemVer Version Scheme. SemVer in this case would be `MAJOR.MINOR`. Where MAJOR will be changed on every change of the document. And MINOR on fixes of document.

Reply:
We might do `MAJOR.MINOR.PATCH` too. Where MAJOR will be major change to document. MINOR will be small change for example some section added which would be big breaking consistency change. PATCH would be fix on document.

Reply:
Added`MAJOR MINOR. PATCH`